### ISSUE TITLE
C2Sense Onboarding – Create Org and Sender Settings in Staging

### ISSUE THREAD

**Issue by MikaelahD13** [CDCgov/prime-reportstream#16383]
### Acceptance Criteria 
- [x] Added to organization.yml and create yamlitos  
    - [ ] Customer status = testing  
    - [ ] For HL7 senders, schemaName: hl7/hl7-ingest-covid-19-training  
    - [ ] For CSV senders, schemaName: upload-covid-19
    - [ ] For CSV OTC senders, schemaName: csv-otc-covid-19

- [x] Added to staging settings table  
- [x] Key exchange  
    - [ ] 1. Shared secret  
         - [ ] Created API keys in Azure staging   
         - [ ] Created API keys in Azure prod  
         - [ ]  Sent API keys to Sender via Keybase  
     - [x] 2. Token auth  
         - [x] Received public key from sender in staging  
              - [ ] Saved public key in Keybase  
         - [ ] Associated public key with sender settings in staging  

FYI: this ticket can be completed concurrently with “Validate Test Files” ticket for this API sender 

### To Do 
- [x] RS engineer: update this ticket with comments with every interaction with sender 
- [x] RS team member who originates this ticket: add any dependencies, as needed 
- [x] RS team member who originates this ticket: make sure the following are included when creating this Issue
     - [x] Pipelines: Onboarding & Operations (New) 
     - [x] Labels: "Onboarding-ops" and "Sender" 
     - [x] Epics: [select the epic for this API onboarding sender]

**Comment by abiphilipnava** [comment]
Created in Staging , and keys are created.
 "keys": [
    {
      "keys": [
        {
          "e": "*******",
          "kid": "******",
          "kty": "******",
          "n": "********"

The actual values is in the configuration.

However waiting for client testing.

**Comment by abiphilipnava** [comment]
still waiting for client testing

**Comment by chris-kuryak** [comment]
Spillover: Still waiting on transport/Auth info in prod.

**Comment by chris-kuryak** [comment]
Splitting into two tickets: this one is for Staging, and a new ticket is for Production (#17129)

**Comment by chris-kuryak** [comment]
@abiphilipnava @victor-chaparro - I separated out Staging from Production settings. Can this staging ticket be closed now, with the modifications I made?
